avm/res/compute/virtual-machine/tests/e2e/windows.atmg/main.test.bicep

```diff
@@ -0,0 +1,92 @@
+targetScope = 'subscription'
```

Comment:
I think - to test/showcase atmg - it may be enough to have 1 test case, like the linux one. As such, I'd remove this test case.

Reply:
done